nh_eobs_mental_health/models/nh_clinical_wardboard.py

```diff
@@ -0,0 +1,91 @@
+from openerp.osv import orm, osv, fields
```

Comment:
What do you think to dropping the `nh_clinical_` prefix from this file name and others? I think it was me who originally suggesting naming it this way but I've realised that in other parts of the system similar files are just named `wardboard.py` or `settings.py` which I think is better because the `nh_clinical` is kind of redundant.